### PR TITLE
the survDataCheck function with a internal idCreate function

### PR DESCRIPTION
**Pull request by philipperuiz**
this is the survDataCheck function but the ID object is created with an function called "idCreate"

### CODE REVIEW

**Comment by pveber** [comment]
Where's the `idCreate` function defined?

**Comment by philipperuiz** [comment]
because the `idCreate` function is use in some other place it is define in a futur file : `morse-internal`

**Comment by pveber** [comment]
Then please add this file with this function only in this PR.

**Comment by philipperuiz** [comment]
know the function `survDataCheck` is associated to the internal function `idCreate`

File: R/survDataCheck.R [hunk 1]
```diff
@@ -151,14 +150,13 @@ survDataCheck <- function(data, diagnosis.plot = TRUE) {
   }
   # 7 test unique triplet Replicat - conc - time only if survDataCheck is called
   # by survData or reproData
-  # FIXME: replace ID creation by function call
-  ID <- str_c(data[, "replicate"], data[, "conc"], data[, "time"],
-              sep = "_")
-  if (any(duplicated(ID))) {
+  if (any(duplicated(idCreate(data, notime = FALSE)))) {
```
Comment (pveber) [inline R/survDataCheck.R]: The expression `idCreate(data, notime = FALSE)` is repeated 3 times. Please reintroduce an `ID` variable instead.

**Comment by philipperuiz** [comment]
the ID variable is reintroduce

**Pull request merged**